File: app/src/main/java/com/qhy040404/libraryonetap/datamodel/CancelData.java
package com.qhy040404.libraryonetap.datamodel;

import com.google.gson.Gson;

import java.io.Serializable;

public class CancelData {
    Gson gson = new Gson();

    public String getMessage(String returnData) {
        return gson.fromJson(returnData, GsonData.class).message;
    }

    private static class GsonData implements Serializable {
        private String message;
    }
}

File: app/src/main/java/com/qhy040404/libraryonetap/datamodel/CancelData.kt
package com.qhy040404.libraryonetap.datamodel

import com.qhy040404.libraryonetap.constant.GlobalManager.moshi

object CancelData {
    fun getMessage(returnData: String): String {
        return moshi.adapter(CancelDataClass::class.java).fromJson(returnData)?.message!!
    }
}